src/tools/db-upgrade/DbUpgrader.cpp

```diff
@@ -322,33 +403,254 @@ void UpgraderSpace::runPartV1() {
           continue;
         }
         // Generate 2.0 value and index records
-        encodeEdgeValue(
-            partId, reader.get(), newEdgeSchema, newKey, strsvId, edgetype, ranking, strdvId, data);
+        encodeEdgeValue(partId,
+                        reader.get(),
+                        newEdgeSchema,
+                        newKey,
+                        strsvId,
+                        edgetype,
+                        ranking,
+                        strdvId,
+                        edgeData);
         lastSrcVertexId = svId;
         lastEdgeType = edgetype;
         lastRank = ranking;
         lastDstVertexId = dvId;
       }
 
-      if (data.size() >= FLAGS_write_batch_num) {
-        VLOG(2) << "Send record total rows " << data.size();
+      // tag data
+      if (tagData.size() >= FLAGS_write_batch_num) {
```

Comment:
Some code could be unified, here and below when `if (tagData.size() > 0) {`, but it is ok for this tool.

Reply:
yeah, fixed.